src/Microsoft.Health.Dicom.Api/Controllers/ExportController.cs

```diff
@@ -0,0 +1,103 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using EnsureThat;
+using MediatR;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Microsoft.Health.Api.Features.Audit;
+using Microsoft.Health.Dicom.Api.Extensions;
+using Microsoft.Health.Dicom.Api.Features.Filters;
+using Microsoft.Health.Dicom.Api.Features.Routing;
+using Microsoft.Health.Dicom.Core.Configs;
+using Microsoft.Health.Dicom.Core.Extensions;
+using Microsoft.Health.Dicom.Core.Features.Audit;
+using Microsoft.Health.Dicom.Core.Messages.Export;
+using Microsoft.Health.Dicom.Core.Models.Export;
+using Microsoft.Health.Dicom.Core.Web;
+using Microsoft.Health.Operations;
+
+namespace Microsoft.Health.Dicom.Api.Controllers;
+
+/// <summary>
+/// Represents an API controller for export operations.
+/// </summary>
+[ApiVersion("1.0-prerelease")]
```

Comment:
Do we need to support this on prerelease version?

Reply:
Ah you're right! We can probably leave it